scripts/svg-component-generator/templates/iconComponent.nunjucks

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PlotlyIconBase from '{{iconBaseComponentPath}}';
+
+const {{componentName}} = (props) => (
+    <PlotlyIconBase viewBox="{{viewBox}}" {...props}>
+        {{content}}
+    </PlotlyIconBase>
+);
+
+export default {{componentName}};
```

Comment:
really... `.nunjucks` as a file extension..?

Reply:
it was from the generator -- I didn't want to spend too much time changing the structure of how it was made originally

Reply:
i believe nunchucks is a type of js templating language